app/assets/locales/messages_ccodk_default.txt

```diff
@@ -400,6 +400,15 @@ template.malformed.chevron=The template file set for your print job has improper
 cannot.set.template=Could not set your print template
 no.file.browser=Your device does not have a file browser installed. Please download one from the playstore and then try again.
 
+start.recording=Start Recording
+stop.recording=Stop Recording

```

Comment:
It looks like these 2 strings must have accidentally gotten into this PR from the audio PR